contracts/ChildAccount.cdc

```diff
@@ -29,8 +29,13 @@ import MetadataViews from "./utility/MetadataViews.cdc"
 ///
 pub contract ChildAccount {
 
-    // TODO:
-    // - Events based on FLIP discussion
+    pub event AccountAddedAsChild(parent: Address, child: Address)
+    pub event ChildAccountCreatedFromManager(parent: Address, child: Address)
+    pub event AccountCreatedFromCreator(creator: Address?, newAccount: Address)
+    pub event ChildAccountGrantedCapability(parent: Address, child: Address, capabilityType: Type)
+    pub event ChildAccountRemoved(parent: Address, child: Address)
+    pub event ChildAccountManagerCreated()
+    pub event ChildAccountCreatorCreated()
```

Comment:
We should probably think critically about what information we want to include in these events. Maybe child account info and other such information. Anything that could potentially be useful to wallets and apps should probably be included. Also, I don't necessarily think the two empty events are that useful

Reply:
I can see why the empty events aren't very informative. My thought was that we could get an idea of adoption vs. actual use - stats on the number of Managers vs linking events could be insightful from a data analysis & user behavior perspective.

Definitely open to suggestions on content of events. I believe at minimum wallets will want to know when any action related to linking occurs for a user's main and potentially any of their linked accounts. E.g. I'd want my wallet to notify me if one of my child accounts had a linking event. And, assuming we maintain Capability granting, events related to granting/removing capabilities from child accounts could be useful to wallet providers. Did you notice any gaps in behavior that aren't covered by the added events?

Reply:
we can keep the empty events if you think they are useful. I'll leave it up to you for suggesting contents, but I would basically include everything you can that is related to the two accounts

Reply:
I think they are useful and agree that more information should be there. Maybe the account which these events are emitted for?

Also I noticed there's no event for the removal of a child account capability in `removeCapability` which is different from the `addCapability` case

Reply:
Events updated based on [feedback](https://github.com/onflow/flips/pull/72#discussion_r1129581219) received in the FLIP.

To your point @franklywatson, since the `createChildAccountManager()` and `createChildAccountCreator()` methods are public contract methods, there's not any information that I can gather we could include in `ChildAccountManagerCreated`/`ChildAccountCreatorCreated` events. Also added an event for revocation of child account Capabilities for consistency as you pointed out.